Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -254,7 +254,7 @@ void WebXROpaqueFramebuffer::endFrame()
         auto gCGL = static_cast<GraphicsContextGLCocoa*>(&gl);
         if (m_ioSurfaceTextureHandleIsShared) {
 #if !PLATFORM(IOS_FAMILY_SIMULATOR)
-            gCGL->detachIOSurfaceFromSharedTexture(m_ioSurfaceTextureHandle);
+            gCGL->detachExternalImage(m_ioSurfaceTextureHandle);
```

Comment:
I believe if you're now abstracting the call, you want to invoke it through gl and not through gCGL.